### PR TITLE
Kmean cluster labels

### PR DESCRIPTION
**Pull request by rogerkuou**
Fix two issues: #57 and #52.

For issue #52, the K-mean label was added as an array with the shape of un-refined clusters. The value at location (band, row, column) represents the refined cluster label of the corresponding band/row/column cluster combination.

- [x] Mention empty clusters in tri-clustering doc.
- [x] Add K-mean labels as results
- [x] Update tests
- [ ] Update tutorial

### CODE REVIEW

**Comment by rogerkuou** [comment]
Hi @fnattino, would you like to review this? I think I would better update the tutorial after we merge this.

File: cgc/kmeans.py [hunk 1]
```diff
@@ -15,6 +15,12 @@ class KmeansResults(Results):
 
     :var k_value: Optimal K value (value with maximum Silhouette score).
     :type k_value: int
+    :var km_labels: Refined clusters labels. It is a 2D- (for colustering)
```
Comment (fnattino) [inline cgc/kmeans.py:18]: Small typo:
```suggestion
    :var km_labels: Refined clusters labels. It is a 2D- (for coclustering)
```

File: cgc/kmeans.py [hunk 2]
```diff
@@ -15,6 +15,12 @@ class KmeansResults(Results):
...
+                    or 3D- (for triustering) array, with the shape of
```
Comment (fnattino) [inline cgc/kmeans.py:19]:
```suggestion
                    or 3D- (for triclustering) array, with the shape of
```

**Review by fnattino — APPROVED** [review]
Wonderful, @rogerkuou , it looks very nice, thank you so much! Just spotted a couple of typos

**Comment by rogerkuou** [comment]
@fnattino Thanks a lot! I have applied your suggestions and will merge.

**Pull request merged**